.github/workflows/lock_old_threads

```diff
@@ -0,0 +1,23 @@
+name: 'Lock Old Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+concurrency:
+  group: lock
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v3
+        with:
+          issue-inactive-days: '7'
```

Comment:
Do you want to increase this limit to ley's say 30 days?

Reply:
There is already a waiting period before a stale issue is automatically closed. I don't think we need to wait much longer to lock it.